launch_openpilot.sh

```diff
@@ -1,3 +1,5 @@
 #!/usr/bin/bash
 
+DISABlE_FW_CACHE=1
```

Comment:
should we just remove the cache? do unplugged DSU people still need it?

Reply:
I believe so, yes